test/Integration/IntegrationTestBase.cs

```diff
@@ -171,7 +182,7 @@ protected void StartFunctionHost(string functionName)
         private string GetFunctionsCoreToolsPath()
         {
             // Determine npm install path from either env var set by pipeline or OS defaults
-            string nodeModulesPath = Environment.GetEnvironmentVariable("node_modules_path");
+            string nodeModulesPath = Environment.GetEnvironmentVariable("NODE_MODULES_PATH");
```

Comment:
We already have logic for determining the path below - why do you need to do this too?